src/loggers.rs

```diff
@@ -27,7 +27,9 @@ where
     logger: L,
     std_err_enabled: bool,
     host_name: String,
+    #[allow(dead_code)]
     service_name: String,
```

Comment:
Can't we just get rid of this instead of needing to #allow dead code on it?

Reply:
Was in 2 minds about this - remove it or leave it in with the annotation. Probably just cleaner to take it out. Addressed.